ray_on_golem/reputation/updater.py

```diff
@@ -0,0 +1,84 @@
+from contextlib import contextmanager
+from typing import Iterable, Tuple
+
+import aiohttp
+
+from ray_on_golem.reputation import models as m
+
+REPUTATION_SYSTEM_URI = "https://reputation.dev-test.golem.network/v1/"
+REPUTATION_SYSTEM_PROVIDER_SCORES = "providers/scores"
+
+
+class ReputationUpdaterException(Exception):
+    ...
+
+
+class ReputationUpdater:
+    """Simple service updating the reputation data from the remote Reputation System."""
+
+    def __init__(self, network: str = "polygon"):
+        self._network = network
+
+    @property
+    def reputation_uri(self):
+        return f"{REPUTATION_SYSTEM_URI}{REPUTATION_SYSTEM_PROVIDER_SCORES}?network={self._network}"
```

Comment:
Use `yarl.URL` here instead of the raw string also, this looks like more of a static value that could be calculated once on init.

Reply:
I somehow don't feel that's much more readable but okay :)